index.js

```diff
@@ -0,0 +1,115 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const models = require('./models')
+
+const app = express()
+
+app.get('/movies', async (request, response) => {
+  const matchingmovies = await models.movies.findAll({
+    include: [{ model: models.directors }, { model: models.genres }]
+  })
+  return response.send(matchingmovies)
+})
+
+app.get('/movies/:identifier', async (request, response) => {
+  const matchingMovies = await models.movies.findOne({
+    where: {
+      id: request.params.identifier,
+    },
+    include: [{ model: models.directors }, { model: models.genres }]
+
+  })
+  if (matchingMovies) {
+    return response.send(matchingMovies)
+  } else {
+    return response.sendStatus(404)
+  }
+})
+
+app.get('/directors/:identifier', async (request, response) => {
+  const matchingDirector = await models.directors.findAll({
+    include: {
+      model: models.movies
+    },
+    where: {
+      id: request.params.identifier
+    }
+  })
+  if (matchingDirector) {
+    return response.send(matchingDirector)
+  } else {
+    return response.sendStatus(404)
+  }
+})
+
+app.get('/genres/:genresName', async (request, response) => {
+  const matchingGenres = await models.genres.findAll({
+    include: {
+      model: models.movies
+    },
+    where: {
+      genresName: request.params.genresName
+    }
+
+  })
+  if (matchingGenres) {
+    return response.send(matchingGenres)
+  } else {
+    return response.sendStatus(404)
+  }
+
+})
+
+app.post('/movies', bodyParser.json(), async (request, response) => {
+  const { title, directors, releaseDate, ratings, runTime, genres } = request.body
```

Comment:
Here you have `ratings` but in the example given in the README it should be rating. This leads to an error when I try to use the example to test.

![image](https://user-images.githubusercontent.com/5775716/70648102-5d664e00-1c18-11ea-85f2-4a359d5f5eeb.png)